puppet/modules/slave/manifests/postgresql.pp

```diff
@@ -1,22 +1,38 @@
 # @api private
 class slave::postgresql {
-  Postgresql_psql {
-    cwd => '/',
-  }
+  # only CentOS slaves are used to run unit tests
+  if $::osfamily == 'RedHat' {
+    if ::operatingsystemmajrelease == '7' {
+      include scl
```

Comment:
Why is this needed? The whole point of the `-syspaths` packages is that you don't need to explicitly enable them.

Reply:
At least in my test VM SCLs are not enabled by default, so nothing can get installed.

Reply:
Perhaps it's easier to just install `centos-release-scl-rh` then to get the repos?

Reply:
I think I need to test all that also on RHEL first, as we do have the AWS slaves on RHEL.

Reply:
On RHEL I think it's a separate subscription. Not entirely sure.